app_utils/sections/experiment.py

```diff
@@ -1774,17 +1774,15 @@ async def experiment_push_to_huggingface_dialog(q: Q, error: str = ""):
         repo_id = f"{user_id}/{exp_name}"
 
         # push tokenizer to hub
-        tokenizer.push_to_hub(
-            repo_id=repo_id,
-            private=True,
-        )
+        tokenizer.push_to_hub(repo_id=repo_id, private=True, safe_serialization=True)
```

Comment:
I think it will just be ignored here, so let's probably remove to avoid confusion,